package.json

```diff
@@ -14,9 +14,20 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.18.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
```

Comment:
Also, this too.

Reply:
ESLint throws an error when "enzyme-adapter-react-16" is mentioned as a devDependency, that's why made it a dependency!